### ISSUE TITLE
Provide typescript typings for linaria/react

### ISSUE THREAD

**Issue by vladmelnikov** [callstack/linaria#275]
**Do you want to request a *feature* or report a *bug*?**
🐞 bug

**What is the current behavior?**
import { styled } from 'linaria/react'; throw typescript error - Could not find a declaration file for module 'linaria/react'. 'C:/Baristika-dashboard/node_modules/linaria/react.js' implicitly has an 'any' type.
  Try `npm install @types/linaria` if it exists or add a new declaration (.d.ts) file containing `declare module 'linaria/react';`

typescript version - 3.2.1
linaria version  -1.0.0-beta.0

**Comment by satya164** [comment]
Hmm. We have types: https://github.com/callstack/linaria/blob/master/types/index.d.ts

Any idea why it's not working? @Esemesek

**Comment by vladmelnikov** [comment]
seems if place file react.d.ts in root directory with current typings its gonna work

**Comment by Esemesek** [comment]
Yup. Typings are there but they are not linked properly. I am working to fix this issue RN.
